src/main/java/net/dries007/tfc/common/blocks/wood/TFCCeilingHangingSignBlock.java

```diff
@@ -20,14 +22,20 @@
 import net.dries007.tfc.common.blocks.ExtendedProperties;
 import net.dries007.tfc.common.blocks.IForgeBlockExtension;
 
-public class TFCCeilingHangingSignBlock extends CeilingHangingSignBlock implements IForgeBlockExtension, EntityBlockExtension
+public class TFCCeilingHangingSignBlock extends CeilingHangingSignBlock implements IForgeBlockExtension, EntityBlockExtension, ITFCHangingSignBlock
 {
     private final ExtendedProperties properties;
+    private final ResourceLocation metal;
 
-    public TFCCeilingHangingSignBlock(ExtendedProperties properties, WoodType type)
+    public TFCCeilingHangingSignBlock(ExtendedProperties properties, WoodType type, ResourceLocation metal)
```

Comment:
AFAICT you are mapping a resource location to a lookup of the `Metal` in `Metal.MANAGER` and then just using the metal's path? You should construct the full model and texture location in the ctor of this class and use it directly in the renderer. For ex. in `JavelinItem` that's what we do. We can also use a convenience ctor that takes a `Metal.Default` that directs to  a constructor that takes a `ResourceLocation` for maximum code cleanliness